src/drafts/MarkdownEditor/_SavedReplies.tsx

```diff
@@ -54,7 +54,7 @@ export const SavedRepliesButton = () => {
       (reply, i): Item => ({
         text: reply.name,
         description: reply.content,
-        descriptionVariant: 'block',
+        descriptionVariant: 'inline',
```

Comment:
Checked on storybook, `inline` doesn't render inline 🤔 but as a block anyway?

![image](https://github.com/primer/react/assets/1863771/139279d9-a4e4-4f57-bc84-b41e796142e5)

Reply:
No, this is inline rendering (which uses Truncate). If you set it to block, the descriptions will take up as much space as they need.
<img width="592" alt="Screenshot 2023-05-11 at 10 21 51 AM" src="https://github.com/primer/react/assets/1825422/a8ca8bd2-bdcd-4b7a-969b-950fe1a0d1db">

Reply:
Yeah, that's what I ended up doing, but just in the case of `_SavedReplies`, since that's the only place that behavior was always occurring.